slack-base/src/main/java/com/hubspot/slack/client/methods/SlackMethods.java

```diff
@@ -156,6 +156,7 @@
   usergroups_users_list(MethodWriteMode.READ, RateLimitingTiers.TIER_2, JsonStatus.FORM_ENCODING_ONLY),
   usergroups_users_update(MethodWriteMode.WRITE, RateLimitingTiers.TIER_2, JsonStatus.ACCEPTS_JSON),
 
+  users_conversations(MethodWriteMode.WRITE, RateLimitingTiers.TIER_3, JsonStatus.FORM_ENCODING_ONLY),
```

Comment:
@darcatron really? a new method doesn't support json? :suspect:

Reply:
Yeah, it's unfortunate: https://api.slack.com/methods/users.conversations
![api encoding warning](https://user-images.githubusercontent.com/5615725/45512374-91c93100-b76d-11e8-8d8f-2d27bcd5d647.png)

Reply:
Conversations are still new things to me (get off my laaaaaawwwwnnnnnn) -- but oh well.